Source/WebKit/UIProcess/API/Cocoa/_WKWebsiteDataStoreDelegate.h

```diff
@@ -60,4 +67,5 @@ WK_API_AVAILABLE(macos(10.15), ios(13.0))
 - (void)websiteDataStore:(WKWebsiteDataStore *)dataStore navigateToNotificationActionURL:(NSURL *)url;
 - (void)requestBackgroundFetchPermission:(NSURL *)mainFrameURL frameOrigin:(NSURL *)frameURL  decisionHandler:(void (^)(bool isGranted))decisionHandler;
 - (void)notifyBackgroundFetchChange:(NSString *)backgroundFetchIdentifier change:(WKBackgroundFetchChange)change;
+- (void)websiteDataStore:(WKWebsiteDataStore *)dataStore domain:(NSString *)registrableDomain didOpenDomainViaWindowOpen:(NSString *)openedRegistrableDomain andAccessProperty:(WKWindowProxyProperty)property directly:(BOOL)directly;
```

Comment:
I would change `andAccessProperty:` to `withAccessProperty:`.  It's very rare to use `and` in an Objective-C method signature.

Reply:
Maybe I'll just go with `withProperty:` then.